Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,611 @@
+```
+
+and
+
+```
+"The multi-platform configuration tool CMake was used for
+ configuring ITK and facilitating its use from our project.
+ CMake was partially funded by the U.S. National Library of
+ Medicine as part of the Insight Toolkit project. CMake is an
+ open source system and it is freely available at www.cmake.org."
+```
+
+### Is ITK FDA-Approved?
+
+Given the fact that ITK is a software toolkit, it cannot be the subject
+of FDA approval as a medical device. We have discussed this topic in
+several occasions and received advice from FDA representatives, that can
+be summarized as follows:
+
+```
+"ITK is to be considered as an off-the-shelf (OTS) product that
+ is used for supporting a higher level medical application/product.
+ The developer of such application/product will be responsible for
+ performing the validation processes described in FDA published
+ guidelines for the development of software-related medical devices."
+```
+
+For mode details see the page [FDA Guidelines for Software Developement](https://itk.org/Wiki/FDA_Guidelines_for_Software_Development).
+
+## Language bindings
+
+### What does `Error opening hint file NOTFOUND` mean?
+
+When you enable `USE_VTK` in the CMake configuration of ITK, and VTK
+was built with wrapping, CMake looks for a "hints" file.
+
+The "hints" file on the VTK source tree is found at: https://gitlab.kitware.com/vtk/vtk/-/blob/master/Common/Core/vtkWrappingHints.h
+
+Please switch CMake to `Advanced`, look for the `VTK_WRAP_HINTS`
+variable and set it to the full path to the "hints" file including the
+`"hints" filename itself.
+
+Here is how it looks in the `CMakeCache.txt` of a typical ITK
+installation:
+
+```
+//Where can the hints file be found
+VTK_WRAP_HINTS:FILEPATH=/home/johndoe/src/VTK/Wrapping/hints
+```
+
+### How do I use ITK with WrapITK?
+
+WrapITK allows you to use ITK from other languages such as Python, Java, or
+Ruby). See the page about [ITK wrappings](https://itk.org/Wiki/ITK/Wrapping).
+
+### How do I use ITK from C# .NET?
+
+There are currently three options:
+
+1. Use [ManagedITK](https://code.google.com/archive/p/manageditk/)
+   - **NOTE: This option is not recommended as the project is no
+     longer supported**.
+1. Use the [SimpleITK](https://simpleitk.org/) C# language
+   bindings:
+   - SimpleITK is great, but it does not support all filters.
+   - A list of available filters can be found
+     [here](http://www.itk.org/SimpleITKDoxygen/html/Filter_Coverage.html)
+     (valid Sept 2012).
+1. Manually wrap the desired functionality
+   - This option requires the most work, but is the most flexible,
+     allowing you to use any ITK functionality you desire
+   - An example can be found [here](https://itk.org/Wiki/ITK/Using_ITK_from_.NET).
+
+NOTE: `WrapITK` does not currently support C# language bindings, but could
+be made to do so.
+
+## Using the Insight Toolkit
+
+### How do I combine ITK and VTK in my application?
```

Comment:
This is out-of-date and could be removed.

Reply:
Removed in 894e5e6.

Reply:
Still seems to be present in the latest build previews.

Reply:
My bad, Matt. Realized now I did not remove it. Done in 52d18a7 PR #4351.